doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1034 @@
+    * The minimal value of the :kconfig:option:`CONFIG_NRF_MODEM_LIB_SHMEM_RX_SIZE` Kconfig option to meet the requirements of modem firmware 1.3.4.
+    * The :c:func:`nrf_modem_lib_diag_stats_get` function now returns an error if called when the :ref:`nrf_modem_lib_readme` library has not been initialized.
+    * The trace backend interface to be exposed to the :ref:`modem_trace_module` using the :c:struct:`nrf_modem_lib_trace_backend` struct.
+    * The :ref:`modem_trace_module` to support backends that store the trace data for later retrieval.
+    * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_RTT` Kconfig option, enabling the RTT trace backend, now requires the :kconfig:option:`CONFIG_USE_SEGGER_RTT` Kconfig option to be enabled.
+
+* :ref:`lib_location` library:
+
+  * Added:
+
+    * Support for the application to send the Wi-Fi access point list to the cloud.
+    * :kconfig:option:`CONFIG_LOCATION_SERVICE_EXTERNAL` Kconfig option that replaces the following configurations:
+
+      * ``CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL``
+      * ``CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL``
+      * ``CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL``
+
+      The new configuration also handles Wi-Fi positioning.
+    * Several new Kconfig options for default location request configurations, including default method priority configuration.
+      These new Kconfig options are applied when the :c:func:`location_config_defaults_set` function is called.
+
+  * Updated:
+
+    * Neighbor cell measurements and Wi-Fi scan results are combined into a single cloud request.
+      This also means that cellular and Wi-Fi positioning are combined into a single cloud positioning method if they are one after the other in the method list of the location request.
+      Because of this, some parts of the API are replaced with new ones as follows:
+
+      * Event :c:enum:`LOCATION_EVT_CLOUD_LOCATION_EXT_REQUEST` replaces old events ``LOCATION_EVT_CELLULAR_EXT_REQUEST`` and ``LOCATION_EVT_WIFI_EXT_REQUEST`` that are removed.
+      * Function :c:func:`location_cloud_location_ext_result_set` replaces old functions ``location_cellular_ext_result_set`` and ``location_wifi_ext_result_set`` that are removed.
+      * Member variable :c:var:`cloud_location_request` replaces old members ``cellular_request`` and ``wifi_request`` that are removed in :c:struct:`location_event_data`.
+      * :kconfig:option:`CONFIG_LOCATION_SERVICE_CLOUD_RECV_BUF_SIZE` replaces ``CONFIG_LOCATION_METHOD_CELLULAR_RECV_BUF_SIZE`` and ``CONFIG_LOCATION_METHOD_WIFI_REST_RECV_BUF_SIZE``.
+
+    * The :ref:`lib_multicell_location` library is deprecated.
+      Relevant functionality from the library is moved to this library.
+      The following features were not moved:
+
+      * Definition of HTTPS port for HERE service, that is :kconfig:option:`CONFIG_MULTICELL_LOCATION_HERE_HTTPS_PORT`.
+      * HERE v1 API.
+      * nRF Cloud CA certificate handling.
```

Comment:
I would move this information to the top of this library subsection (line 748), as it is not just a simple line about an update. Also, I would add a link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/software_maturity.html#api-deprecation (see comment below).